apps/sample-ui/src/index.tsx

```diff
@@ -0,0 +1,29 @@
+import './index.css'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { RecoilRoot } from 'recoil'
+import App from './App'
+import reportWebVitals from './reportWebVitals'
+
+const queryClient = new QueryClient()
+if (['development', 'test'].includes(process.env.NODE_ENV)) {
+  const { worker } = require('./mocks/browser')
+  worker.start()
```

Comment:
I want to have a separate package and start mock service worker there rather than starting it here, but I need to see if that's possible 🤔

Reply:
I think it's fair enough that we expect the frontend application to start the worker like this. same way we need to hook up the redux store and initialise the react query client. I think as-is is fine